TeachingRecordSystem/src/TeachingRecordSystem.AuthorizeAccessToATeacherRecord/Infrastructure/FormFlow/DummyCurrentUserIdProvider.cs

```diff
@@ -0,0 +1,8 @@
+using TeachingRecordSystem.SupportUi.Infrastructure.FormFlow;
+
+namespace TeachingRecordSystem.AuthorizeAccessToATeacherRecord.Infrastructure.FormFlow;
+
+public class DummyCurrentUserIdProvider : ICurrentUserIdProvider
```

Comment:
This is temporary, until we have a way of assigning a session ID cookie that we can use in place of a user ID.